### DISCUSSION TITLE
Solana

### DISCUSSION THREAD

**Discussion by GoldTaro** [Huniko-Team/community#191] [Q&A] [answered]
What is Proof of History (PoH) and how does it contribute to Solana's performance?

**Comment by Tvenus** [comment, marked as answer]
Proof of History is a cryptographic clock that provides a timestamp for each transaction, creating a historical record that proves events occurred in a specific sequence. This innovation reduces the overhead and time required for transaction validation, contributing to Solana's high throughput and efficiency.

**Comment by GoldTaro** [comment]
Thank you
